classes/person.rb

```diff
@@ -0,0 +1,23 @@
+class Person
+  attr_accessor :name, :age
+  attr_reader :id
+
+  def initialize(age, name = 'Unknown', parent_permission: true)
+    @id = nil
+    @name = name
+    @parent_permission = parent_permission
+    @age = age
+  end
+
+  private
+
+  def of_age?
+    @age >= 18
+  end
+
+  public
+
+  def can_use_services?
+    of_age? && @parent_permission
+  end
```

Comment:
- [ ] [OPTIONAL] You can start with this method `can_use_services?` and you don't have to add a public flag